source/overview/product.rst

```diff
@@ -22,47 +22,50 @@ Core committers, including both community contributors and paid staff at Matterm
-
-Details of each offering are as follows:
-
-**Mattermost Team Edition** - A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source "virtual office", offering all the core productivity benefits of competing SaaS solutions. It deploys as single Linux binary with MySQL or PostgreSQL under an MIT license. Features include:
-
-- One-to-one and group messaging, file sharing and unlimited search history
+- One-to-one and group messaging, file sharing, and unlimited search history
 - Native apps for iOS, Android, Windows, Mac, Linux
-- Threaded messaging, emoji and custom emoji
+- Threaded messaging, emoji, and custom emoji
 - Tools for custom branding
-- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__ and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
+- Continuous archiving
+- Multi-factor authentication
+- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__, and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
 - Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`__ and `third party extensions <https://about.mattermost.com/default-app-directory/>`__
 - Easily scalable from dozens to hundreds of users
-- New improvements released every two months
-- Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, Spanish, and Ukrainian.
+- New features and improvements released every two months
 
 To get started, `download the open source Mattermost Team Edition server <https://about.mattermost.com/download>`__ under an MIT license.
 
-**Mattermost Enterprise Edition E10** is a commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
+Mattermost Enterprise Edition E10
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
```

Comment:
Should we include something like "building on the features offered in Team Edition"? We have a similar line in the E20 description and wondering if we either include that here as well or remove it from both descriptions? If we remove it then a link to here: https://mattermost.com/pricing-feature-comparison/ at the start of the section would work.

Reply:
Yes I think we can -- but we could also link to the feature comparison page in the intro (I'm 0/5)